AdvancedTask/AdvancedTask/AssertHelpers/SkillAssertion.cs

```diff
@@ -0,0 +1,71 @@
+﻿using AdvancedTask.Pages.Components.ProfileOverview;
+using AdvancedTask.Test_Model;
+using AdvancedTask.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AdvancedTask.AssertHelpers
+{
+   public class SkillAssertion:BaseClass
+    {
+        SkillMethodComponents SkillMethodComponentsObj;
+        public SkillAssertion()
+        {
+            SkillMethodComponentsObj = new SkillMethodComponents();
+        }
+        public void AssertAddedSkill()
+        {
+            List<Skill> SkillData = JsonReader.ReadTestDataFromJson<Skill>("A:\\Industry Connect\\AdvancedSprint1\\AdvancedTask\\AdvancedTask\\Json Test Data\\AddSkill.json");
+
+            
+            IWebElement SkillRecord = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[3]/div/div[2]/div/table/tbody/tr/td[1]"));
```

Comment:
your assestions should not contain any web logic it should have the information already here to here, this information needs to be return as part of a step where the page action method would do and return this please fix this everywhere thanks

Reply:
Done